.prow/config.yaml

```diff
@@ -146,7 +146,7 @@ presubmits:
     always_run: true
     spec:
       containers:
-      - image: maven:3.6-jdk-11
+      - image: gcr.io/kf-feast/feast-ci:latest
```

Comment:
Can you explain these changes?

Reply:
This provides the test with `go` executable that it needs to run `go test`.

Reply:
Reverted.